utils/alternative_case_sens_fix

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Can you please remove this file?  And/or do it in a cross-platform way?  I am striving for platform-independency in all parts of this repo.